hassio/src/dialogs/network/dialog-hassio-network.ts

```diff
@@ -316,7 +316,7 @@ export class DialogHassioNetwork
       >
         <div class="radio-row">
           <ha-formfield
-            .label=${this.supervisor.localize("dialog.network.dhcp")}
+            .label=${this.supervisor.localize("dialog.network.auto")}
```

Comment:
And then you need to update the translation of the legacy supervisor UI too 😄 
Under `supervisor.dialog.network`